### DISCUSSION TITLE
Unload combine CP + AD

### DISCUSSION THREAD

**Discussion by VoldBart** [Courseplay/Courseplay_FS22#2580] [Q&A]
Hi!
I have a problem with unloading combine using CP and AD. I have selected everywhere option to unload combine at 80% but it doesn't work. Combine still goes till it reaches 95%, stops, unfold the pipe and then calls for AD to unload it. 
It is huge waste of time especially on large fields. I tried several options and it didn't work for me.
I'm also wondering that I cannot find option in CP "Always unfold the pipe". I'm guessing that it might help if I turned it to activated.
I'm using AD v2.0.1.0 and CP 7.2.1.4

Thanks for all the help!

**Comment by nitroSkirata** [comment]
For Unloading to work with CP combine and AD unloader, you must set the fillevel at wich the combine shall call for the unloader in the AD settings of the combine.

The Combine will at that level open the pipe and call the unloader.

**Reply by Tensuko** [reply]
Correct, its AD Setting if you use AD as Unloader.

**Comment by nitroSkirata** [comment]
Best would be to take screenshots of the setting pages for us to look over

**Comment by VoldBart** [comment]
**CP Harvester:**
![cp_combain1](https://github.com/Courseplay/Courseplay_FS22/assets/119314344/2a1a6354-a7cf-4cb2-8f53-073d745e1a91)
![cp_combain2](https://github.com/Courseplay/Courseplay_FS22/assets/119314344/4532e1c9-229a-4638-94fd-a15a34979312)
![cp_combain3](https://github.com/Courseplay/Courseplay_FS22/assets/119314344/6489cb0d-1583-4a93-ac83-a15e9bcc821d)

**AD Harvester:**
![ad_combain1](https://github.com/Courseplay/Courseplay_FS22/assets/119314344/c3815abd-4c6e-4ef9-b209-84f192256512)
![ad_combain2](https://github.com/Courseplay/Courseplay_FS22/assets/119314344/9fe0f096-877f-41ee-a7bd-948900ab91fb)
![ad_combain3](https://github.com/Courseplay/Courseplay_FS22/assets/119314344/85b716cb-dd87-41a9-8a22-5fcb66e3fcd2)

**CP Unloader:**
![cp_unloader1](https://github.com/Courseplay/Courseplay_FS22/assets/119314344/a526b05e-e9d7-4ac2-b480-96a0362ac930)
![cp_unloader2](https://github.com/Courseplay/Courseplay_FS22/assets/119314344/7d7aa58f-5f38-437e-8a9c-db5a189834af)

**AD Unloader:**
![ad_unloader_1](https://github.com/Courseplay/Courseplay_FS22/assets/119314344/54a82709-ec89-4913-a03f-5997a99a130b)
![ad_unloader_2](https://github.com/Courseplay/Courseplay_FS22/assets/119314344/6719333b-5149-49ba-a4a8-56b972a0f65b)

**HUD Harvester:**
![ad2](https://github.com/Courseplay/Courseplay_FS22/assets/119314344/521ba2f6-8f90-40db-a11f-770b27161a6d)

**HUD Unloader:**
![ad1](https://github.com/Courseplay/Courseplay_FS22/assets/119314344/f6f408c9-11e6-4f2b-a775-3da4007bb813)

**Reply by nitroSkirata** [reply]
ok, as it is setup, the Combine will request the unloader to come to it, when it reaches 80% fill level, as set in the Combine AD settings, pre call level.

If you want the combine to call ealyer, you have to reduce that setting.

**Reply by VoldBart** [reply]
This is the issue I'm having. Harvester reaches 80% (as it is set) fill level but it does not call for unload (it does not unfold the pipe). Harvester still goes until reaches 95% and it stops. Only then unloader is coming. I don't know where is the issue...

**Reply by nitroSkirata** [reply]
in that case, best open an issue at the AD github. that problem is on them, not Courseplay.

https://github.com/Stephan-S/FS22_AutoDrive

**Comment by rohne83** [comment]
AD is not chasing combines as soon as they are above 80 or 90% filllevel. Noone knows why, but currently that's what AD dev guys have implemented. There are various issues about this in AD github, so before opening a new one, rather comment in existing one.
CP unloaders always chase for your 80% filllevel. As you're using CP for your harvester you're better off using CP for unloader as well.

**Reply by coatsy35** [reply]
Is it possible to use CP to unload combine and then have autodrive take over to delivery to production or silo? if so how do you set this up? currently I have AD unload on combine mode which delivers to silo, but I'd like to try CP as unloader due to the above 80/90% issue with AD

**Reply by Tensuko** [reply]
Read the ingame help or check www.courseplay.dev
